APPLICATION_PROGRESS.md

```diff
@@ -61,6 +61,42 @@ The information in this section is split into subsections in order to conceptual
 
 ---
 
+<details>
+<summary>Automated Market Maker (AMM)</summary>
+
+<h3>Contracts ✅</h3>
+
+- Feature complete for UI integration
+  - Needs vec support in SDK so that array can be changed to vec
```

Comment:
Vec support? I thought that this was implemented some time ago? Do we need to bump the repo for this (we need to bump anyway)?

Reply:
That's actually an issue for the scripts, I don't remember what I was thinking when I wrote that down to the contracts section. Removed it and will have to bump for scripts anyway, #294